decompiler/IR2/FormExpressionAnalysis.cpp

```diff
@@ -660,7 +660,7 @@ void SimpleExpressionElement::update_from_stack_identity(const Env& env,
       result->push_back(x);
     }
   } else if (arg.is_static_addr()) {
-    auto lab = env.file->labels.at(arg.label());
+    auto& lab = env.file->labels.at(arg.label());
```

Comment:
can you make these `const auto&`s? Just in case some of this code used to modify the copy and would cause problems if it modified the original?